core/deployment/src/main/java/io/quarkus/deployment/index/ApplicationArchiveBuildStep.java

```diff
@@ -247,18 +247,6 @@ private static boolean containsMarker(Path dir, Set<String> applicationArchiveFi
     }
 
     private static Index handleFilePath(Path path) throws IOException {
```

Comment:
I assume this change along with the one `AppModelGradleResolver` have been vetted by @aloubyansky ?

Reply:
Yes